From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 21 Sep 2025 16:17:12 +0200
Subject: [PATCH] Don't remove PATH from tests

A recent GDK-PixBuf with Glycin will attempt to spawn sandboxed loaders,
which needs `bwrap` from the PATH.
---
 scripts/isolate-test.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/isolate-test.sh b/scripts/isolate-test.sh
index 82dadd145428..9aafcac0416f 100755
--- a/scripts/isolate-test.sh
+++ b/scripts/isolate-test.sh
@@ -77,7 +77,7 @@ mkdir -p "$XDG_CONFIG_HOME"
 export G_DEBUG="fatal-critical"
 
 echo "Variables in isolated environment:" >&2
-env -i G_DEBUG="$G_DEBUG" HOME="$HOME" XDG_CONFIG_HOME="$XDG_CONFIG_HOME" XDG_RUNTIME_DIR="$XDG_RUNTIME_DIR" dbus-run-session -- env >&2
+env -i G_DEBUG="$G_DEBUG" HOME="$HOME" PATH="$PATH" XDG_CONFIG_HOME="$XDG_CONFIG_HOME" XDG_RUNTIME_DIR="$XDG_RUNTIME_DIR" dbus-run-session -- env >&2
 echo >&2
 
-env -i G_DEBUG="$G_DEBUG" HOME="$HOME" XDG_CONFIG_HOME="$XDG_CONFIG_HOME" XDG_RUNTIME_DIR="$XDG_RUNTIME_DIR" dbus-run-session -- "$@"
+env -i G_DEBUG="$G_DEBUG" HOME="$HOME" PATH="$PATH" XDG_CONFIG_HOME="$XDG_CONFIG_HOME" XDG_RUNTIME_DIR="$XDG_RUNTIME_DIR" dbus-run-session -- "$@"
